README.md

```diff
@@ -47,31 +44,26 @@ _Or_
 
 _Or_
 
- * Use the environment variable
+ * Use the environment variable in the shell where you start jupyter lab
 
    ```
-   setenv FIREFLY_URL http://localhost:8080/firefly
+   export FIREFLY_URL=http://localhost:8080/firefly
    ```
 
 **where the URL points to a Firefly server.**
```

Comment:
This is just the difference between `tcsh` and `bash` it would be better:
```
in bash:
   export FIREFLY_URL=http://localhost:8080/firefly
in tcsh: 
   setenv FIREFLY_URL http://localhost:8080/firefly

Reply:
Definitely agree with using the bash syntax -- it's the POSIX standard shell.

Reply:
OK, I'll add it for tcsh too